sdk/hdinsight/Microsoft.Azure.HDInsight.Job/src/generate.ps1

```diff
@@ -1 +1 @@
-Start-AutoRestCodeGeneration -ResourceProvider "hdinsight/data-plane" -AutoRestVersion "latest"
\ No newline at end of file
+Start-AutoRestCodeGeneration -ResourceProvider "hdinsight/Microsoft.Azure.HDInsight.Job/src" -AutoRestVersion "latest"
```

Comment:
What is the intent behind these changes? `-ResourceProvider` corresponds to the relative path to `specifications` folder in REST specs repo. The paths being introduced here don't even exist

Reply:
I see what you mean, I have removed those changes from the PR. It seems the .cmd files need to be updated still?